proxy/papyrus/Dockerfile

```
@@ -0,0 +1,10 @@
+FROM ghcr.io/starkware-libs/papyrus:dev
+
+USER root
+
+RUN apk update && apk add --no-cache ca-certificates
```

Hadolint

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)